communities/epfl-hackers/community-assessment.md

```diff
@@ -0,0 +1,87 @@
+- Historically, has the group always been about the same size? If not, what brought on that change?
+	- Can't really tell as there has been no metric. Many students are working on cool tech projects, but there hasn't really been any gathering to form a group.
+
+### Resources
+- What is currently available to people in your community to learn about tech? Are there meetups, seminars, online tools, office hours, hackathons?
+	- Outside of lectures, not alot.
+	- I organzied the first hackathon at EPFL last year - LauzHack
+	- There are colloquiums organized by the CS department at EPFL where guest lecturers present their research followed by a Q&A, although these are not very frequent and have fairly low attendance.
+	- Around "recruiting season" several companies come to EPFL and have some talks and workshops.
+
+### Accessibility
+- How accessible are the current resources in your community?
+	- Quite accessible - pretty much anyone can join in.
+- Are meetings held during hours where people may be working?
+	- There are no meetings held at the moment, although its something we're looking to start. If we do, it will be in the evenings when people aren't working/in class.
+
+### Interests
+- Is your community made up only of computer science students? Are you OK with that?
+	- It contains students who study Computer Science or Communication Systems, but we're totally open to students from all areas in EPFL.
+- Is your community open to other topics, or are their interests very specific?
+	- It is quite open to suggestion and will accomodate any topic as long as it is tech related, and there is interest in it.
+
+### Goals
+- Why do people get involved in your community?
+	- Its an opportunity to work on cool project where they have the freedom to work on anything they are intersted in, rather than something they are forced to in their classwork. Its also a support structure as there are other people to guide you if you are stuck or to give you advice - this way you aren't doing this on your own.
+- Are they students hoping to get a degree?
+	- Yes
+- Are they trying to ramp-up to get a job?
+	- Yes, but not all of them
+- Are they passionate about developing, regardless of their major?
+	- Yes
+- What activities and goals can you set for your community based on their existing goals?
+	- Goals
+		- Learn something new
+		- Work on a 'side-project' on your own/with friends
+	- Activities
+		- Participate in hackathons
+		- Have occasional meetups to discuss anything (questions, challenges, doubts about a project, a class, an interview)
+		- Have regular workshops to (usually) learn/practice and (occasionally) teach something.
+
+### Location
+- Where does your community currently gather? Is it an online-only group so far, or is there a place where people can meet?
+	- So far we just talk online - we aren't big enough to have "gatherings". But if/when we meet there are lots of public spaces and easily accessible classrooms at EPFL's campus.
```

Comment:
I guess what I meant was that since we are just a few friends we don't have official gatherings, rather we just meet each other as friends and also talk about tech. But certainly if we are official we'll have gatherings regardless of size.